_includes/team_header.html

```diff
@@ -1,54 +1,68 @@
+  <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image offset{{iter}}"
+    title="{{ person.first_name }} {{ person.last_name }}"
+    src="http://www.centonovelosgatos.com/wp-content/uploads/2013/08/unknown-female.jpg"
+    onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
   {% endif %}
-  </a>
-  {% assign iter = iter | plus: 1 | modulo: 2 %}
-{% elsif person.role == "postdoc" or person.role == "phdstudent" or person.role == "phd_rotation" or person.role == "msstudent" or person.role == "undergrad" or person.role =="intern" or person.role =="visitor" or person.role =="engineer"%}
-  <a class="personKey {{ person.key}}" href="{{ person.website }}"><!--   onclick="selectPerson('{{ person.key }}')">-->
+</a>
+{% assign iter = iter | plus: 1 | modulo: 2 %}
+{% elsif person.role == "postdoc" or person.role == "phdstudent" or person.role == "phd_rotation" or person.role ==
+"msstudent" or person.role == "undergrad" or person.role =="intern" or person.role =="visitor" or person.role
+=="engineer"%}
+<a class="personKey {{ person.key}}" href="{{ person.website }}">
+  <!--   onclick="selectPerson('{{ person.key }}')">-->
   {% if person.image %}
-    <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}" title="{{ person.first_name }} {{ person.last_name }}" src="{{ site.baseurl }}{{ person.image }}" onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
+  <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}"
+    title="{{ person.first_name }} {{ person.last_name }}" src="{{ site.baseurl }}{{ person.image }}"
+    onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
   {% else %}
-    <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}" title="{{ person.first_name }}  {{ person.last_name }}" src="http://www.centonovelosgatos.com/wp-content/uploads/2013/08/unknown-female.jpg" onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
+  <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}"
+    title="{{ person.first_name }}  {{ person.last_name }}"
+    src="http://www.centonovelosgatos.com/wp-content/uploads/2013/08/unknown-female.jpg"
+    onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
   {% endif %}
-  </a>
-  {% assign iter = iter | plus: 1 | modulo: 2 %}
+</a>
+{% assign iter = iter | plus: 1 | modulo: 2 %}
```

Comment:
Is everything above this a change to formatting?

Reply:
yes formatting